### DISCUSSION TITLE
Coapp no longer works on MacOS 10.13

### DISCUSSION THREAD

**Discussion by jadenquinn** [aclap-dev/video-downloadhelper#288] [Help and Q&A]
### Your question

I use MacOS 10.13 specifically for many reasons, updating or downgrading causes issues for every computer I own, from every year of machine that can run above 10.13, so I am on this version and will never update on the machines that I intend to use. These are reasonable powerful machines, Quad core i7s and such, and they are extremely usable. I have 9 machines on 10.13, approximately, and that's the newest that those machines have. I'm typing this on one of them.

The problem is... well the coapp now just refuses to register with the Firefox (or whatever browser you use) extention anymore. It used to work. I have no idea what changed, as I didn't exactly update anything other than Firefox itself. I use Firefox because it's the fastest browser, no other reason. It just is.

Will this be fixed, or is there a fix I am not aware of? I know that 10.13 has specific issues related to plugins for existing stuff, for example Audacity's FFMPEG addon thing refuses to link up with Audacity, for example, and it's ALWAYS annoying.

### Addon & CoApp version and details

Latest on Firefox, and the coapp is v1 right now that I found somewhere on this forum that someone suggested might work, as the v2 installer doesn't work on 10.13, even though the installer opens and everything, it just fails to install.

**Comment by mec008** [comment]
Same problem here

**Reply by stolenfyre** [reply]
Thank you kit529738. Worked for me. Closed firefix. Had to delete the old coapp. reinstalled the versions you specified. Ran the coapp from the applications files. Opened firefox. Now all is good. Again, thank you.

**Comment by kit529738** [comment]
Hi- I'm using an older mac air on High Sierra-- I cannot update my system past 10.13. 

I've been able to continue using both helper and coapp by installing the older version of both the extension and the app, 

[Firefox DL Helper extension version 8.0.0.6](https://addons.mozilla.org/firefox/downloads/file/4157053/video_downloadhelper-8.0.0.6.xpi)

[CoApp version 1.6.3](https://github.com/aclap-dev/vdhcoapp/releases/download/v1.6.3/net.downloadhelper.coapp-1.6.3.pkg)

Go to ffox extension settings and TURN OFF automatic updates, this will preserve the version
I recommend deleting all old versions of both the coapp and extension and then RELOAD the extension and open the coapp from your applications once installed. this will prompt the two to link up again. 

Once i had those correctly installed, it did take a minute (and a few refreshes) for the extension to correctly recognize the media file on the webpage I was on, but then it downloaded with no issues!  

![Screen Shot 2024-03-31 at 2 04 18 PM](https://github.com/aclap-dev/video-downloadhelper/assets/165589299/cdfa7fe2-fa77-4e60-b375-a25fd2b9362b)

**Reply by Knei2ser** [reply]
Sorry!
Old Version installation works,
CoApp doesn't :-(
![downloadhelper coapp-1 6 3](https://github.com/aclap-dev/video-downloadhelper/assets/173044397/914c04cc-3821-48ac-9861-fdc190653378)

**Reply by jadenquinn** [reply]
This doesn't work. Error:
BootyLooty15:~ jaden$ /Applications/net.downloadhelper.coapp.app/Contents/MacOS/register.sh ; exit;
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/./vdhcoapp (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/./vdhcoapp (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

**Reply by carnzoid** [reply]
Doesn't work me neither, Companion app doesn't start up, High Sierra 10.13.6 .

**Reply by sbgraphic** [reply]
Hi, 

I had the same  issue on 10.13.x but i finnally was able to resolve it this way:

– After you're [installation complete ](https://github.com/aclap-dev/video-downloadhelper/discussions/288#discussioncomment-8966571) quit Firefox…
– Go to your /applications folder and try to open "net.downloadhelper.coapp"
– Go to "System preferences / security settings and authorize this application [see apple help](https://support.apple.com/en-au/guide/mac-help/mh40616/10.13/mac/10.13)
– Open firefox and go to extensions/Video DownloadHelper/preferences… Companion app should now be found and you can start using Video DownloadHelper…

Best to all

**Comment by LigmaJohnsonn** [comment]
Heij!
I created my account just to tell Y´all that the downloadhelper-8.0.0.6. with the coapp-1.6.3. did it for me. I run mac 10.13 and struggled with this from the beginning.
Maybe I just had the wrong versions combo- dunno… it works now.
It would be nice if the coapp give a confirmation after opening it after install. It is very irritating to me that I click it and seemingly nothing happened. A little something, a thumbs up pop up- something like that.
Otherwise I have -now that it works- nothing to complain.
Exept for the planned obsolescense. That bugger made me "upgrade"- it was Firefox that fucked me sideways!
On my old Imac I didn´t recognize that FF is basically a zombie. Sync didn´t work no more- I ran version 78 or so.
And then I fucked up: I deleted what I thought was the cache but it was something else and I was fucked.
I barely recovered, had to buy a "new" computer just to run a newer Firefox. Dissappointing.
Anyways… I have everything working now, I even got me a Time Capsule. Tested it and worked flawless.
That´s why I love the old Apple stuff, the Steve Universe.
Have a nice day, with Kind Regards
